apps/src/javalab/JavalabConsole.jsx

```diff
@@ -130,20 +167,9 @@ class JavalabConsole extends React.Component {
             ref={el => (this._consoleLogs = el)}
             className="javalab-console"
           >
-            <div style={styles.logs}>{this.displayConsoleLogs()}</div>
-            <div style={styles.lineWrapper}>
-              <InputPrompt onClick={this.focus} />
-              <input
-                type="text"
-                spellCheck="false"
-                style={{
-                  ...styles.input,
-                  ...(isDarkMode ? styles.darkMode : styles.lightMode)
-                }}
-                onKeyDown={this.onInputKeyDown}
-                aria-label="console input"
-              />
-            </div>
+            <label style={styles.logs} htmlFor="console-input">
```

Comment:
will wrapping in a label cause any accessibility issues?

Reply:
Great question.  I digested a bunch at https://stackoverflow.com/questions/774054/should-i-put-input-elements-inside-a-label-element.  It was valid to put an `input` inside a `label`, and I did that so that we could put the input on the same line as preceding output text.  This is usually an implicit association which wouldn't require a `for` attribute on the `label`, however I read that some screen readers prefer to have that, so I added it.  However, it turns out a `label` should not have any `divs` inside of it, so I reworked the console to no longer use a `label` at all.  Now, we get the same behaviour as with the `label`, but with an explicit click handler on the log text which sets the focus to the `input`.

I tested with macOS VoiceOver before this PR, with the `label` approach, and with the new `div`, and behaviour appears to be consistent, though I think there could be room for improvement with future work.